boards/adafruit-itsybitsy-m4/include/gpio_params.h

```diff
@@ -42,5 +42,5 @@ static const  saul_gpio_params_t saul_gpio_params[] =
 }
 #endif
 
-#endif /* GPIO_PARAMS_H */
 /** @} */
+#endif /* GPIO_PARAMS_H */
```

Comment:
Here specifically the `@{` and the `@}` would now be unbalanced, as one of them is within and the other outside of the part covered by the include guard. I'm nit sure if I am just pedantic, or if this really could cause issues.

I think we wouod be better off with `#pragma once`.

Reply:
Yeah, as mentioned in the initial message, that's the case everywhere.


> I think we wouod be better off with `#pragma once`.

I tend to agree. Let's maybe discuss `#pragma once` and the required coding convention changes next friday (I've already added it to the agenda).

Reply:
I second this. This unbalance was the first thing to come to my mind, upon opening this PR.

Also, I would be in support of a move to `#pragma once`. I moved to it years ago. It is just one less thing to think about with each new header. Reduction of cognitive load is always nice. And no worries of the include guard name colliding if you have headers of the same name in different locations, or when files get renamed, on top of all the other benefits.